tool/intellij/README.md

```diff
@@ -0,0 +1,36 @@
+## Live Templates for MobX
+
+### Add Live Templates
+
+Copy file `MobX.xml` to:
```

Comment:
I think here you can create two sections, one for Android and other for Intellij. I think we have both kinds of users.

Reply:
Rest looks great

Reply:
I don't know the paths to Intellij.
You want me to make an AndroidStudio folder and an Intellij folder ?